### PR TITLE
Refactor config map and secrets

### PR DESCRIPTION
**Pull request by lhein**
needs #136

### CODE REVIEW

File: src/CamelKNodeProvider.ts [hunk 1]
```diff
@@ -168,58 +161,63 @@ export class CamelKNodeProvider implements vscode.TreeDataProvider<TreeNode> {
...
 	// retrieve the list of integrations running in camel k using the kube proxy and rest API
 	getIntegrationsFromCamelKRest(): Promise<Object> {
-		return new Promise( async (resolve, reject) => {
+		return new Promise( (resolve, reject) => {
 			let proxyURL = utils.createCamelKRestURL();
-			await utils.pingKubernetes().catch( (error) =>  {
-				reject(error);
-			});
-			var options = {
+			const options = {
```
Comment (lhein) [inline src/CamelKNodeProvider.ts]: need to change that to ```let``` instead

Comment (lhein) [reply src/CamelKNodeProvider.ts]: fixed in previous PR for Node Provider Refactoring

**Comment by bfitzpat** [comment]
> now, the "Refreshing Apache Camel K Integrations view succeeded, no published integrations available." is logged several times in the Apache Camel K output channel. it is happening even when there are inegration published.

@apupier This has been fixed when you look at the combined PR https://github.com/camel-tooling/vscode-camelk/pull/141. I ran into the same issue when trying to test these separately. There's something about the combination of changes that corrects the multiple message issue

**Comment by bfitzpat** [comment]
Closed with merging of combined PR https://github.com/camel-tooling/vscode-camelk/pull/141

**Pull request closed without merging**